src/lib/i18n.js

```diff
@@ -0,0 +1,38 @@
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+
+const resources = {
+  'en-US': {
+    translation: {
+      searchBar: {
+        placeholder: 'Search',
+      },
+    },
+  },
+  'en-GB': {
+    translation: {
+      searchBar: {
+        placeholder: 'Search',
+      },
+    },
+  },
+  pa: {
+    translation: {
+      searchBar: {
+        placeholder: 'ਖੌਜ',
+      },
+    },
+  },
+}
+
+i18n.use( initReactI18next ).init( {
+  resources,
+  lng: 'en-US',
+  fallbackLng: 'en-US',
+  debug: true,
+  interpolation: {
+    escapeValue: false,
+  },
+} )
+
+export default i18n

```

Comment:
I'm considering that it may be better to keep the strings with the components (or even within), just like we do styles.

Reply:
I think we'd have to use a different solution than i18next if we want to inline them

Reply:
https://www.i18next.com/how-to/add-or-load-translations

We can do this.